cmd/mailbox-controller/controller.go

```diff
@@ -62,6 +63,8 @@ type mbCtl struct {
 	queue                   workqueue.RateLimitingInterface // of mailbox workspace Name
 }
 
+var initialSuffix uint64 = 142857
```

Comment:
But this _variable_ does not hold the _initial_ suffix in general, its job is to hold the suffix.

Reply:
Changed in latest push.